src/lib/sdl_renderer.cpp

```diff
@@ -44,15 +49,25 @@ namespace aronnax {
 
   void SDLRenderer::drawRectangle(const Vector2d& pos,
                                   const Vector2d& box,
-                                  const Color& c={ 0, 0, 0, 0 })
+                                  SDL_Texture* texture,
+                                  float angle
+                                  )
   {
     SDL_Rect r;
-    r.x = int(pos.x);
-    r.y = int(pos.y);
+    r.x = int(pos.x - box.x / 2);
```

Comment:
Should this type of physics to rendering conversion be done by the caller of this function?

It should be SDL specific, so probably not.